aiven/client/cli.py

```diff
@@ -1750,6 +1750,73 @@ def service__schema__subject_version__delete(self):
         project_name = self.get_project()
         self.client.delete_schema_subject_version(project_name, self.args.name, self.args.subject, self.args.version_id)
 
+    @arg.project
+    @arg.service_name
+    def service__mirrormaker__replication_flow__list(self):
```

Comment:
Having `service mirrormaker` makes sense since it's service level commands, but for consistency and save a word from a long command it could be worth having the mirrormaker commands right on the top level so `mirrormaker replication-flow list` without the `service` at the start. Compare with eg. `kafka topic state`

Reply:
Makes sense, changed